### DISCUSSION TITLE
Debonding interval length

### DISCUSSION THREAD

**Discussion by expyri911** [oasisprotocol/community-forum#3] [Ideas]
Hi, i think the debonding is really expensive ... may can you turn off ? it's really bad idea the debonding waiting time ...
Thx for you help !

https://mgt-corp.fr/

**Comment by tjanez** [comment]
Hi @expyri911,

> i think the debonding is really expensive ... may can you turn off ? it's really bad idea the debonding waiting time ...

are trying to say that the current [debonding inverval](https://docs.oasis.dev/general/mainnet/genesis-file#staking-and-rewards) of 336 epochs (approximately 14 days) to reclaim staked ROSE tokens is too long?

**Comment by expyri911** [comment]
> Hi @expyri911,
> 
> > i think the debonding is really expensive ... may can you turn off ? it's really bad idea the debonding waiting time ...
> 
> are trying to say that the current [debonding inverval](https://docs.oasis.dev/general/mainnet/genesis-file#staking-and-rewards) of 336 epochs (approximately 14 days) to reclaim staked ROSE tokens is too long?

Yes really too long, some people want undelegate and restake immediatly but they can't. i think it's better for oasis to remove this option !! :)

**Comment by expyri911** [comment]
i have an idea, may propose 2 options, 
1/ with unstake immeditaly but with fees and the fees are give to the node
2/ Reduce debonding to 7 days or less

thx

**Comment by vycl87** [comment]
In my opinion, debonding time can shorten the time a bit to be more binding on the Oasis community. In this way, the community can feel in a position to take action more easily.

**Comment by expyri** [comment]
yes pls so long ... pls reduce this or revome ...

**Comment by jevinoh** [comment]
I agree with @expyri911 option no. 1 where if anyone wants to unstake immediately, a fee will be given to node.
(Maybe a percentage of the total rewards received)

14 days is a very long time to take action (especially if someone want to take some losses)

**Comment by godyolops** [comment]
hey man, debonding of rose is too long. we should have implement that option to unstake immediately at a cost of rose/fee. this 2 weeks aint good man! I have lots of people in my community, How can endorse staking if the debonding is 2 weeks. omg!
